src/dev/flang/fuir/analysis/dfa/Call.java

```diff
@@ -322,12 +324,36 @@ public String toString()
       .append(r == null ? "*** VOID ***" : r);
     if (_env != null)
       {
-        sb.append(_env.toString());
+        sb.append(" ENV: ");
+        sb.append(dev.flang.util.Terminal.INTENSE_PURPLE + _env.toString() + dev.flang.util.Terminal.RESET);
```

Comment:
I think `Errors.effe` could be used here as well since it is already used in this file below.